src/filing/actions/updateSignature.js

```diff
@@ -6,11 +6,18 @@ import requestSignaturePost from './requestSignaturePost.js'
 import { postSignature } from '../api/api.js'
 import { error } from '../utils/log.js'
 
-export default function updateSignature(signed) {
+const MAX_SIGNING_ATTEMPTS = 10
+
+export default function updateSignature(signed, attempts = 0) {
   return dispatch => {
     dispatch(requestSignaturePost())
```

Comment:
As part of this temp fix we could also condition dispatching the `requestSignaturePost()` behind whether we're on the first attempt or not (since if we aren't we've already dispatched it and the store is updated appropriately).

`if(!attempts) dispatch(requestSignaturePost())`